build/controllers/ReleaseController.php

```diff
@@ -874,7 +874,7 @@ protected function splitChangelog($file, $version)
                 $state = 'end';
             }
             // add continued lines to the last item to keep them together
-            if (!empty(${$state}) && trim($line !== '') && strpos($line, '- ') !== 0) {
+            if (!empty(${$state}) && trim($line) !== '' && strncmp($line, '- ', 2) !== 0) {
```

Comment:
This one looks serious.

Reply:
A little bit, but due to weak typing this worked. I just noticed that we can drop `trim($line) !== ''` here at all, should I do it?